README.md

```diff
@@ -11,157 +18,79 @@ Client:
 
 Server:
 
-- Node.js,
+- Node,
 - Express.js
 
 Database:
 
 - MongoDB
 
-Tools:
-
-- Webpack,
-- Eslint (custom rules extends Airbnb),
-- Prettier,
-- Husky (pre-commits hooks).
-- Docker
-
-### Seeding database for development
-
-At first run of the app, log in and check your user id in the database under eoc/users directory. Put that user ID, generated by mongo to .env file under USER_ID variable and save the file.
-Then simply run `npm run seed`.
-Seeding is not working? Probably USER_ID in your database is not matching with USER_ID in your .env file. Be aware that if you clean up the whole database with users collection, your USER_ID delivered by mongo will be different compared to the previous session. **TIP** When playing with data, always clean all the collections omitting users collection, so you don't have to deal with `USER_ID` configuration over and over again.
-
-### Google Client ID and Client Secret Key
-
-Before running the app remember to add the Google client id key, Client secret key and Express session key to .env file.
+All of the used technologies you can find in package.json files.
```

Comment:
I would go for 'libraries'/'packages' wording instead of technologies when it comes to package.json.